tests/dragonfly/snapshot_test.py

```diff
@@ -500,3 +502,33 @@ async def test_tiered_entries_throttle(async_client: aioredis.Redis):
 
     await load_task
     assert await StaticSeeder.capture(async_client) == start_capture
+
+
+@dfly_args({"proactor_threads": 1, "dbfilename": "big_snapshot"})
+async def test_big_value_serialization_memory_limit(df_factory):
```

Comment:
Ideally, we would have a version of this test per each supported data type
Would that be hard?

Reply:
not really, I will make them part of this test. It's a few more lines -- I will just be explicit -- no fancy abstractions :nerd_face: